File: poo_java/Gato.java
class Gato extends Animal{
    protected String tamaniobigotes;

    public Gato(Persona persona,String nombre,String tamaniobigotes){
       super(persona,nombre);
       this.tamaniobigotes = tamaniobigotes;
    }
    //polimorfismo
    @Override
    public String getNombre() {
        return "polimorfismo " +this.nombre;
      }

}

File: poo_js/gato.js
class Gato extends Animal{
    tamaniobigotes;

    constructor(persona,nombre,tamaniobigotes){
        super(persona,nombre);
        this.tamaniobigotes = tamaniobigotes;
    }

    getTamanioBigotes(){
        return this.tamaniobigotes;
    }
    getNombre(){
        return  "polimorfismo "  + this.nombre;
    }
}